### PR TITLE
Updated input manifests

### PR DESCRIPTION
**Pull request by prudhvigodithi**
Signed-off-by: Prudhvi Godithi <pgodithi@amazon.com>

### Description
Updated input manifests for 2.6.0
Creating a manual PR until the issue https://github.com/opensearch-project/opensearch-build/issues/3068 is addressed

### Issues Resolved
N/A

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: manifests/2.6.0/opensearch-2.6.0.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+schema-version: '1.0'
+build:
+  name: OpenSearch
+  version: 2.5.0
```
Comment (gaiksaya) [inline manifests/2.6.0/opensearch-2.6.0.yml:5]: version should be 2.6.0

File: manifests/2.6.0/opensearch-dashboards-2.6.0.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+schema-version: '1.0'
+build:
+  name: OpenSearch Dashboards
+  version: 2.5.0
```
Comment (gaiksaya) [inline manifests/2.6.0/opensearch-dashboards-2.6.0.yml:5]: version should be 2.6.0

**Review by gaiksaya — CHANGES_REQUESTED** [review]

File: manifests/2.6.0/opensearch-2.6.0.yml (hunk 1 above)
Comment (prudhvigodithi) [reply manifests/2.6.0/opensearch-2.6.0.yml]: Thanks @gaiksaya human errors :P

**Review by gaiksaya — APPROVED** [review]

**Review by zelinh — APPROVED** [review]

**Review by saratvemulapalli — APPROVED** [review]

**Comment by saratvemulapalli** [comment]
3.0.0 manifest is failing, looks like we split up the repos?
```
2023-01-10 21:41:21 ERROR    CI Manifest check failed
Traceback (most recent call last):
  File "./src/run_ci.py", line 25, in <module>
    sys.exit(main())
  File "./src/run_ci.py", line 20, in main
    CiManifests.from_file(args.manifest, args).check()
  File "/home/runner/work/opensearch-build/opensearch-build/src/ci_workflow/ci_manifest.py", line 22, in check
    self.__check__()
  File "/home/runner/work/opensearch-build/opensearch-build/src/ci_workflow/ci_input_manifest.py", line 42, in __check__
    ci_check_list.check()
  File "/home/runner/work/opensearch-build/opensearch-build/src/ci_workflow/ci_check_list_source.py", line 44, in check
    instance = klass(self.component, self.git_repo, self.target, check.args)
  File "/home/runner/work/opensearch-build/opensearch-build/src/ci_workflow/ci_check_gradle_properties.py", line 20, in __init__
    self.properties = self.__get_properties()
  File "/home/runner/work/opensearch-build/opensearch-build/src/ci_workflow/ci_check_gradle_properties.py", line 35, in __get_properties
    return PropertiesFile(self.git_repo.output(cmd))
  File "/home/runner/work/opensearch-build/opensearch-build/src/git/git_repository.py", line 80, in output
    return subprocess.check_output(command, cwd=cwd, shell=True).decode().strip()
  File "/opt/hostedtoolcache/Python/3.7.15/x64/lib/python3.7/subprocess.py", line 411, in check_output
    **kwargs).stdout
  File "/opt/hostedtoolcache/Python/3.7.15/x64/lib/python3.7/subprocess.py", line 488, in run
    with Popen(*popenargs, **kwargs) as process:
  File "/opt/hostedtoolcache/Python/3.7.15/x64/lib/python3.7/subprocess.py", line 800, in __init__
    restore_signals, start_new_session)
  File "/opt/hostedtoolcache/Python/3.7.15/x64/lib/python3.7/subprocess.py", line 1551, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpeukd5ura/dashboards-reports/reports-scheduler': '/tmp/tmpeukd5ura/dashboards-reports/reports-scheduler'
```

**Comment by gaiksaya** [comment]
The CI is failing for reports-scheduler. Looks like `working-directory` is not required anymore https://github.com/opensearch-project/opensearch-build/blob/main/manifests/3.0.0/opensearch-3.0.0.yml#L50
@rupal-bq Please confirm. 
Thanks!

**Comment by zelinh** [comment]
Seems like we have removed the front end code and split the repo in this PR for main branch. https://github.com/opensearch-project/reporting/pull/630
Then working_directory would need to be removed.

**Comment by rupal-bq** [comment]
> The CI is failing for reports-scheduler. Looks like `working-directory` is not required anymore https://github.com/opensearch-project/opensearch-build/blob/main/manifests/3.0.0/opensearch-3.0.0.yml#L50 @rupal-bq Please confirm. Thanks!

Yes, I merged repo split changes so `working-directory` isn't required now.

**Comment by prudhvigodithi** [comment]
The workflow still fails with error
```
FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpj2e28g59/dashboards-reports/reports-scheduler': '/tmp/tmpj2e28g59/dashboards-reports/reports-scheduler'
```

**Comment by gaiksaya** [comment]
I believe it needs a rebase? @prudhvigodithi

**Pull request merged**